apps/dsb-message-broker/src/message/message.query.pipe.ts

```diff
@@ -3,8 +3,24 @@ import { PipeTransform, Injectable, BadRequestException } from '@nestjs/common';
 @Injectable()
 export class MessageQueryPipe implements PipeTransform<any> {
     async transform(data: any) {
-        if (!data.fqcn) throw new BadRequestException('fqcn is missing in the query!');
-        if (typeof data.fqcn !== 'string') throw new BadRequestException('fqcn must be a string!');
+        if (data.from) {
+            if (!isNaN(data.from) || isNaN(Date.parse(data.from))) {
+                throw new BadRequestException(
+                    'The value of from parameter is not a valid dateTime.'
+                );
+            }
+
+            if (data.from) {
+                if (data.from.includes('.')) {
```

Comment:
Is that causes some sort of errors?

Reply:
...by default Date().toISOString() returns date with ms, so we are asking users to make extra step, so I there a reason?

Reply:
in ISO format milliseconds are after dot and durable name for consumer cannot have dots. i fixed in next commit by replacing all dots with underscores.